functions/tab.xfce4_terminal.fish

```diff
@@ -0,0 +1,3 @@
+function tab.xfce4_terminal -a cdto
+  xfce4-terminal --tab --working-directory "$cdto"
```

Comment:
I have some ideas; working on it. It's pretty straightforward to make it work in bash, but I'm just starting w/ fish.